services/getNavigation.js

```diff
@@ -1,17 +1,60 @@
 import { getDocsListBySection } from ".";
 
+const cache = new Map();
+
 export const getNavigation = async () => {
-  const foundationDocs = await getDocsListBySection("foundations");
-  const componentDocs = await getDocsListBySection("components");
-  return [
-    {
-      sortItems: true,
-      category: "Foundations",
-      docs: foundationDocs,
-    },
-    {
-      category: "Components",
-      docs: componentDocs,
-    },
-  ];
+  let navData = [];
+
+  if (cache.has("nav")) {
+    console.log("cache hit: getNavigation");
+    navData = cache.get("nav");
+  } else {
+    const foundationDocs = await getDocsListBySection("foundations");
+    const componentDocs = await getDocsListBySection("components");
+
+    // filter out content from docs
+
+    const foundationDocsList = foundationDocs.map((doc) => {
+      const { data, content, slug, filePath } = doc;
+
+      return {
+        data: {
+          title: data?.title || "",
+          order: data?.order || null,
+        },
+        slug,
+        filePath,
+      };
+    });
+
+    const componentDocsList = componentDocs.map((doc) => {
+      const { data, content, slug, filePath } = doc;
+
+      return {
+        data: {
+          title: data?.title || "",
+          order: data?.order || null,
```

Comment:
Wen the Nav builds the status indicators are missing from the nav. Need to include status here?

Reply:
Just pushed up a fix.

Reply:
yep I saw that, but it's up on line 24 for Foundations. looks like need one here as well